eslint.config.js

```diff
@@ -15,23 +16,12 @@ module.exports = [
   ...compat.extends(
     'not-an-aardvark/node',
     'plugin:eslint-comments/recommended',
-    'plugin:node/recommended',
     'plugin:prettier/recommended',
     'plugin:unicorn/recommended'
   ),
+  pluginN.configs['flat/recommended'],
   {
-    languageOptions: { sourceType: 'commonjs' },
     rules: {
-      'comma-dangle': [
```

Comment:
`comma-dangle` is removed as: 1) we are using prettier now; 2) formatting rules are deprecated.